src/applications/caregivers/containers/IntroductionPage.jsx

```diff
@@ -0,0 +1,213 @@
+              </li>
+            </ul>
+            <div>
+              <h5>What if I need help filling out my application?</h5>
+              <p>
+                An accredited representative, like a Veterans Service Officer
+                (VSO), can help you fill out your claim. {''}
+                <a href="/disability-benefits/apply/help/index.html">
+                  Get help filing your claim.
+                </a>
+              </p>
+
+              <CaregiverSupportInfo />
+            </div>
+          </li>
+          {/* Apply */}
+          <li className="process-step list-two">
+            <h5>Apply</h5>
+            <p>
+              Complete this Caregiver benefits form or have a Power of
+              Attorney/Representative complete form. You’ll get a confirmation
+              message after submitting the form. You can print this for your
+              records.
+            </p>
+            <PowerOfAttorneyInfo />
+            <RepresentativeInfo />
+            <span>You can Also:</span>
+            <ul>
+              <li>Call us at 877-222-VETS (877-222-8387)</li>
+              <li>
+                Find a <a href="/#">Caregiver Support Coordinator</a>
+              </li>
+              <li>Contact the Nation Caregiver Support line at 855-260-3274</li>
+              <li>Contact a Veterans Service Organization</li>
+            </ul>
+          </li>
+          {/* Review */}
+          <li className="process-step list-three">
+            <h5>VA Review</h5>
+            <p>
+              Once the form has been received at a medical center, the Caregiver
+              Support Coordinator (CSC) will review the application and contact
+              the Veteran and Caregiver.
+            </p>
+
+            <strong>
+              If the Veteran initially meets eligibility requirements, the next
+              steps are:
+            </strong>
+
+            <ul>
+              <li>
+                A visit with the Veteran and Caregiver at the medical center
+                where the Veteran Receives Care
+              </li>
+              <li>Caregiver training</li>
+              <li>A home visit</li>
+            </ul>
+          </li>
+          {/* Decision */}
+          <div className="process-step list-four">
+            <h5>Decision</h5>
+            <p>You’ll get a notice in the mail with our decision.</p>
+          </div>
+        </ol>
+      </div>
+    </div>
+  );
+
+  return (
+    <div className="schemaform-intro caregiver-intro-page">
+      <FormTitle title="Apply for Caregiver Benefits" />
+      <p>Equal to VA Form 10-10CG (Application for Caregiver Benefits)</p>
+
+      <IntoHighlight />
+
+      <button
+        className="va-button-link schemaform-start-button"
+        onClick={startForm}
+      >
+        Start your Application
+      </button>
+
+      <Benifits />
+
+      <ProcessTimeline />
+
+      <button style={{ marginBottom: '20px' }} onClick={startForm}>
```

Comment:
Margin bottom is defined here: https://design.va.gov/utilities/margins#margin-bottom

Reply:
I was going to look for it I swear 😆